Zotero/Scenes/Detail/PDF/Views/AnnotationsViewController.swift

```diff
@@ -478,29 +516,34 @@ final class AnnotationsViewController: UIViewController {
 
     private func setupKeyboardObserving() {
         NotificationCenter.default
-                          .keyboardWillShow
-                          .observe(on: MainScheduler.instance)
-                          .subscribe(with: self, onNext: { `self`, notification in
-                              if let data = notification.keyboardData {
-                                  self.setupTableView(with: data)
-                              }
-                          })
-                          .disposed(by: self.disposeBag)
+        .keyboardWillShow
+        .observe(on: MainScheduler.instance)
+        .subscribe(with: self, onNext: { `self`, notification in
+            if let data = notification.keyboardData {
+                self.setupTableView(with: data)
+            }
+        })
+        .disposed(by: self.disposeBag)
```

Comment:
Shouldn't these be moved a tab ahead to be properly aligned?

Reply:
Yes, forgot to indent it, good catch